src/atomate2/aims/schemas/calculation.py

```diff
@@ -0,0 +1,494 @@
+        None, description="The final total DFT energy for the calculation"
+    )
+    energy_per_atom: float = Field(
+        None, description="The final DFT energy per atom for the calculation"
+    )
+    atoms: MSONableAtoms = Field(
+        None, description="The final structure from the calculation"
+    )
+    structure: Union[Structure, Molecule] = Field(
+        None, description="The pymatgen structure of the final Atoms"
+    )
+    efermi: float = Field(
+        None, description="The Fermi level from the calculation in eV"
+    )
+
+    forces: Optional[list[Vector3D]] = Field(
+        None, description="Forces acting on each atom"
+    )
+    all_forces: Optional[list[list[Vector3D]]] = Field(
+        None,
+        description="Forces acting on each atom for each structure in the output file",
+    )
+    stress: Optional[Matrix3D] = Field(None, description="The stress on the cell")
+    stresses: Optional[list[Matrix3D]] = Field(
+        None, description="The atomic virial stresses"
+    )
+
+    is_metal: Optional[bool] = Field(None, description="Whether the system is metallic")
+    bandgap: Optional[float] = Field(
+        None, description="The band gap from the calculation in eV"
+    )
+    cbm: float = Field(
+        None,
+        description="The conduction band minimum, or LUMO for molecules, in eV "
+        "(if system is not metallic)",
+    )
+    vbm: Optional[float] = Field(
+        None,
+        description="The valence band maximum, or HOMO for molecules, in eV "
+        "(if system is not metallic)",
+    )
+    atomic_steps: list[MSONableAtoms] = Field(
+        None, description="Structures for each ionic step"
+    )
+
+    @classmethod
+    def from_aims_output(
+        cls,
+        output: AimsOutput,  # Must use auto_load kwarg when passed
+        store_trajectory: bool = False,
+    ) -> "CalculationOutput":
+        """
+        Create an FHI-aims output document from FHI-aims outputs.
+
+        Parameters
+        ----------
+        output: .AimsOutput
+            An AimsOutput object.
+        store_trajectory: bool
+            A flag setting to store output trajectory
+
+        Returns
+        -------
+        The FHI-aims calculation output document.
+        """
+        atoms = output.final_structure
+
+        electronic_output = {
+            "efermi": output.fermi_energy,
+            "vbm": output.vbm,
+            "cbm": output.cbm,
+            "bandgap": output.band_gap,
+            "direct_bandgap": output.direct_band_gap,
+        }
+
+        forces = None
+        if output.forces is not None:
+            forces = output.forces
+
+        stress = None
+        if output.stress is not None:
+            stress = voigt_6_to_full_3x3_stress(output.stress).tolist()
+
+        stresses = None
+        if output.stresses is not None:
+            stresses = [
+                voigt_6_to_full_3x3_stress(st).tolist() for st in output.stresses
+            ]
```

Comment:
@utf We might need to check units for stresses in general. ase uses eV/Angstrom^3.

Reply:
If I recall correctly someone mentioned that atomate2 uses kbar earlier in the PR so in the parsers I added a unit conversion to move from eV/AA^3 to kbar. FHI-aims also uses eV/AA^3 by default.

Reply:
Great. Thanks. That was me. The force field schema had the wrong unit for stresses. This is why I wanted to make sure.